client/src/pages/Profile/Profile.tsx

```diff
@@ -16,12 +17,15 @@ import ContestList from '../../components/ContestList/ContestList';
 import AuthHeader from '../../components/AuthHeader/AuthHeader';
 import { Contest } from '../../interface/User';
 import { getContestByUser } from '../../helpers/APICalls/contest';
+import updateProfile from '../../helpers/APICalls/profile';
 
 export default function Profile(): JSX.Element {
   const classes = useStyles();
   const [value, setValue] = useState(0);
   const [contests, setContests] = useState<Contest[]>([]);
-  const { loggedInUser } = useAuth();
+  const [selectedFile, setSelectedFile] = useState<File>();
+  const [newProfilePic, setNewProfilePic] = useState<string>();
```

Comment:
Currently we are storing the url in a local state within this component. Can we please find a way to make the profile picture persist on the FE?

Reply:
This is done.